docs/versioning.md

```diff
@@ -0,0 +1,29 @@
+### Assumptions
+This project uses [semver v2](https://semver.org/), as does the rest of OpenTelemetry.
+
+### Goals
+#### API Stability:
+* Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function, with any SDK that has the same major version, and equal or greater minor or patch version.
```

Comment:
```suggestion
* Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function with any SDK that has the same major version and an equal or greater minor version.
```

https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html

patch versions should always be compatible if major/minor are equal